src/test/javascript/spec/component/exam/exam-timer.component.spec.ts

```diff
@@ -40,20 +40,20 @@ describe('ExamTimerComponent', () => {
 
     it('should update display times', () => {
         let duration = dayjs.duration(15, 'minutes');
-        expect(component.updateDisplayTime(duration)).toBe('15:00');
+        expect(component.updateDisplayTime(duration)).toBe('15min');
         duration = dayjs.duration(-15, 'seconds');
-        expect(component.updateDisplayTime(duration)).toBe('00');
+        expect(component.updateDisplayTime(duration)).toBe('0min 0s');
         duration = dayjs.duration(8, 'minutes');
-        expect(component.updateDisplayTime(duration)).toBe('08:00');
+        expect(component.updateDisplayTime(duration)).toBe('8min 0s');
         duration = dayjs.duration(45, 'seconds');
-        expect(component.updateDisplayTime(duration)).toBe('45');
+        expect(component.updateDisplayTime(duration)).toBe('0min 45s');
     });
 
     it('should display exact time', () => {
         let duration = dayjs.duration(629, 'seconds');
-        expect(component.updateDisplayTime(duration)).toBe('10:29');
+        expect(component.updateDisplayTime(duration)).toBe('10min');
         duration = dayjs.duration(811, 'seconds');
-        expect(component.updateDisplayTime(duration)).toBe('13:31');
+        expect(component.updateDisplayTime(duration)).toBe('13min');
     });
```

Comment:
The test case name is no longer correct, right? We want to **hide** the exact time above a certain threshold. Could you update the test case description?